### PR TITLE
Upgrader to Python 3.7

### PR DESCRIPTION
**Pull request by ringsaturn**
Details: https://github.com/caiyunapp/rb/pull/1

******

Can I say rb will not update in the future for sentry will use redis_clusters and drop rb?

https://github.com/getsentry/sentry/blob/247962bc1045791c234cc402f5dd67db8ef296ec/src/sentry/utils/redis.py#L159-L163

```py
# TODO(epurkhiser): When migration of all rb cluster to true redis clusters has
# completed, remove the rb ``clusters`` module variable and rename
# redis_clusters to clusters.
clusters = ClusterManager(options.default_manager)
redis_clusters = ClusterManager(options.default_manager, _RedisCluster)
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +1,42 @@
+dist: xenia
```
Comment (joshuarli) [inline .travis.yml:1]:
```suggestion
dist: xenial
```

File: .travis.yml [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
 
 sudo: false
 
 matrix:
   include:
     - os: linux
-      python: 2.6
+      python: 3.5
     - os: linux
-      python: 2.7
```
Comment (joshuarli) [inline .travis.yml:10]: We'll want to keep testing on 2.7. At a minimum, 2.7 and 3.6.

**Comment by joshuarli** [comment]
Recommended https://github.com/getsentry/rb/pull/27/commits/1aa36e0feab75f6cd89dc0c1323e6981c94d858c.

**Comment by joshuarli** [comment]
Thanks for the changes @ringsaturn, we'll take it from here.

**Comment by cclauss** [comment]
Please rebase or close this PR.

**Comment by ringsaturn** [comment]
All the changes to work under Python3.7 have been made. Our fork has been in our production environment for a year while this PR had opened for a year without any comments from getsentry org.

Hope you guys could figure out a way to ensure rb work under both 2.7 and 3.7, and I don't have time to fight with Python2's str/unicode/byte types because it's too hard :scream: .

Thanks to people who made actual process for the upgrading.

**Pull request closed without merging**